includes/class-wc-calypso-bridge-admin-setup-checklist.php

```diff
@@ -86,12 +86,25 @@ private function handle_redirects() {
 	}
 
 	/**
-	 * Returns if the checklist is "done" via the user clicking the "I'm done" button.
+	 * Returns if the checklist is "done" via the user clicking the "I'm done" button
+	 * or if the store already contains any orders.
 	 *
-	 * @return bool True if the done button has been clicked.
+	 * @return bool
 	 */
 	private function is_checklist_done() {
-		return (bool) get_option( 'atomic-ecommerce-setup-checklist-complete', false );
+		if ( apply_filters( 'wc_calypso_bridge_development_mode', false ) ) {
```

Comment:
As you [noted in your review](https://github.com/Automattic/wc-calypso-bridge/pull/478#discussion_r306224823), should we change this method to public so we can utilize elsewhere?

Reply:
Updated to a static method in https://github.com/Automattic/wc-calypso-bridge/pull/479/commits/01368a411b63751fde6ec8c6920e3f5188ece472